scripts/Muon/GUI/Common/muon_file_utils.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import (absolute_import, division, print_function)
+
+import os
+
+allowed_instruments = ["EMU", "MUSR", "CHRONUS", "HIFI"]
+allowed_extensions = ["nxs"]
+FILE_SEP = os.sep
+
+
+def filter_for_extensions(extensions):
+    """Filter for file browser"""
+    str_list = ["*." + str(ext) for ext in extensions]
+    return "Files (" + ", ".join(str_list) + ")"
+
+
+def get_instrument_directory(instrument):
+    """
+    If instrument is supported, returns the directory name on the ISIS network
+    in which its data can be found
+    """
+    if instrument in allowed_instruments:
+        instrument_directory = instrument
+        if instrument == "CHRONUS":
+            instrument_directory = "NDW1030"
+        return instrument_directory
+    else:
+        return None
+
+
+def check_file_exists(filename):
+    return os.path.isfile(filename)
+
+
+def get_current_run_filename(instrument):
+    """
+    If instrument is supported, attempts to find the file on the ISIS network which
+    contains the data from its current (most up-to-date) run.
+    """
+
+    instrument_directory = get_instrument_directory(instrument)
+    if instrument_directory is None:
+        return ""
+
+    autosave_file_name = _instrument_data_directory(instrument) + FILE_SEP + "autosave.run"
```

Comment:
looking at the `_instrument_data_directory` method it calls `instrument_directory`, which is above. Why can we not pass in the instrument directory?